x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_overview.tsx

```diff
@@ -100,15 +112,27 @@ export const EnginesOverview: React.FC = () => {
           />
         </EuiPageContentBody>
 
-        {metaEngines.length > 0 && (
+        {canViewMetaEngines && (
```

Comment:
See above comment, I would change this to a `hasPlatinumLicense` check:

```suggestion
        {hasPlatinumLicense && (
```